### ISSUE TITLE
`pod lib lint` fails with Code Signing error

### ISSUE THREAD

**Issue by bizz84** [bizz84/SwiftyStoreKit#285]
### Platform

- [ ] iOS
- [x] macOS
- [ ] tvOS

### In app purchase type

- [ ] Consumable
- [ ] Non-consumable 
- [ ] Auto-Renewable Subscription
- [ ] Non-Renewing Subscription

### Environment

- [ ] Sandbox
- [ ] Production

### Version
Latest [`master`](https://github.com/bizz84/SwiftyStoreKit)

### System Information

Running this command:

```
{ sw_vers; echo "---"; xcodebuild -version; echo "---"; echo "Cocoapods version: $(pod --version)"; }
```

yields this:

```
ProductName:	Mac OS X
ProductVersion:	10.12.6
BuildVersion:	16G29
---
Xcode 9.0
Build version 9A235
---
Cocoapods version: 1.3.1
```

### Related issues
https://github.com/CocoaPods/CocoaPods/issues/6451

### Issue summary
On command line, run:
```
pod lib lint --verbose
```

### What did you expect to happen
SwiftyStoreKit passed validation.

### What happened instead
Validation fails with:
```
[!] SwiftyStoreKit did not pass validation, due to 1 error.
You can use the `--no-clean` option to inspect any issue.
```

The build log reveals the two following problems on the macOS build:

```
Code Signing Warning: "App" isn't code signed but requires entitlements. It is not possible to add entitlements to a binary without signing it.
Code Signing Error: Code signing is required for product type 'Application' in SDK 'macOS 10.13'
```

```
   Testing with xcodebuild. 
 -> SwiftyStoreKit (0.11.0)
    - ERROR | [OSX] xcodebuild: Returned an unsuccessful exit code.
    - NOTE  | [OSX] xcodebuild:  ld: warning: both MACOSX_DEPLOYMENT_TARGET and IPHONEOS_DEPLOYMENT_TARGET are set
```

[build.log](https://github.com/bizz84/SwiftyStoreKit/files/1379274/build.log)

The same problem applies when trying to publish a new pod:

```
pod trunk push SwiftyStoreKit.podspec
```

This prevents from publishing a new pod version.

I have tried resetting the code signing settings [as suggested here](https://stackoverflow.com/questions/37806538/code-signing-is-required-for-product-type-application-in-sdk-ios-10-0-stic/39379587) but to no avail.

Any ideas?

**Comment by shuhrat10** [comment]
@bizz84 I just tried clone master and run `pod lib lint --verbose`

** BUILD SUCCEEDED **
 -> SwiftyStoreKit (0.10.8)
SwiftyStoreKit passed validation.

How did you get SwiftyStoreKit (0.11.0) ?

**Comment by bizz84** [comment]
@shuhrat10 which cocoapods version are you using?

Could you paste the result of this?
```
{ sw_vers; echo "---"; xcodebuild -version; echo "---"; echo "Cocoapods version: $(pod --version)"; }
```

**Comment by shuhrat10** [comment]
ProductName: Mac OS X
ProductVersion: 10.13
BuildVersion: 17A362a
---
Xcode 8.3.3
Build version 8E3004b
---
Cocoapods version: 1.2.1

**Comment by bizz84** [comment]
Just tried this again and it worked.

Published [version 0.11.0](https://github.com/bizz84/SwiftyStoreKit/releases/tag/0.11.0).

**Comment by shuhrat10** [comment]
👍
